Document the process of porting over samples to hero sample repositories (#2140)
Azure/communication-ui-library: docs/releases/creating-a-release.md

    ![Trigger the deploy storybook action](../images/trigger-storybook-publish.png)
1. Once everything is deployed and published on npm, delete the release branch from github.

## Step ∞

*For stable release only*, update the hero samples to use the newly released stable package. See [instructions for updating hero samples](./hero-samples.md).

> It is acceptable to update the samples within ~a week of the package release.

Reviewer: nit: Can this be more assertive: 
```suggestion
Samples should be updated within a week of the package release
```
